src/ray/raylet/worker_pool.h

```diff
@@ -39,6 +40,56 @@ namespace raylet {
 using WorkerCommandMap =
     std::unordered_map<Language, std::vector<std::string>, std::hash<int>>;
 
+/// \struct WorkerCacheKey
+///
+/// Struct used to cache workers, keyed by runtime_env. The dynamic_options and
+/// override_environment_variables will be unified into runtime_env in the future.
+struct WorkerCacheKey {
```

Comment:
Why do you use struct? Can we just make it a class?

Reply:
Sounds good, let me make it a class